crates/python/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "qcs-sdk-python"
 description = "Python bindings to qcs-sdk-rust"
-version = "0.1.2"
+version = "0.1.0"
```

Comment:
Not sure if this is intended, but the version number went *down*

Reply:
That was the result of me fussing around with the version numbers to test knope. I reverted back to 0.1.4, which was the latest version to be released on TestPyPI. 

`knope` is bumping to `0.2.0` whether it's going from `0.1.0` [or `0.1.4`](https://github.com/rigetti/qcs-sdk-rust/pull/144/commits/f2f422808dd98742b86c64439c7d3d3ff999f3db). Not sure what rule it's following that is triggering a minor bump here.

With the `knope` configuration we have, versions are tracked separately. For example, if all commits after the previous release are scoped to `(rust)`, then a new release would only bump the Rust crate.

We can choose to track them together. I think the intent was to track them separately since it is possible to make a breaking change in one package and not the other.